src/app/content/highlights/components/SummaryPopup/HighlightsPrintButton.tsx

```diff
@@ -0,0 +1,44 @@
+import flow from 'lodash/fp/flow';
+import React from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components/macro';
+import { AppState, Dispatch } from '../../../../types';
+import PrintButton from '../../../components/Toolbar/PrintButton';
+import { printSummaryHighlights } from '../../actions';
+import { hasMoreResults, summaryIsLoading } from '../../selectors';
+
+interface Props {
+  className: string;
+  shouldFetchMore: boolean;
+  isLoading: boolean;
+  print: typeof printSummaryHighlights;
+}
+
+// tslint:disable-next-line:variable-name
+const HighlightsPrintButton = ({className, print, shouldFetchMore, isLoading}: Props) => {
+  return <PrintButton
+    className={className}
+    onClick={() => print({shouldFetchMore})}
```

Comment:
there is no reason to pass `shouldFetchMore`, you can either:
- use `shouldFetchMore` here to either dispatch the action or just call window.print
- use `hasMoreResults` within the hook to see if more results are needed